CRM/Core/EntityTokens.php

```diff
@@ -662,7 +662,8 @@ protected function addFieldToTokenMetadata(array &$tokensMetadata, array $field,
       // Not the existing QuickForm widget has handling for the custom field
       // format based on the title using this syntax.
       $parts = explode(': ', $field['label']);
```

Comment:
I'm not 100% sure what's going on here, but if you're trying to avoid splitting the colons in the second part, I think this will work
```suggestion
      $parts = explode(': ', $field['label'], 2);
```

Reply:
Oh yes, that's definitely better. I'll rework.